### ISSUE TITLE
[PACKAGE REQUEST] srecord 

### ISSUE THREAD

**Issue by Spinus1** [msys2/MSYS2-packages#1778]
Hello,
it would be very nice to have Srecord tool in msys2: it is a must have to to manipulate file in srecord format (especially useful for embedded devices development)

Upstream project is: http://srecord.sourceforge.net/
There is also an Ubuntu ppa: see http://srecord.sourceforge.net/download.html

I really like to help packaging this tool since it is used in my company, but I don't know where to start :)

BR,
Alessio

**Comment by Biswa96** [comment]
The repository seems to be old. The SourceForge page shows last update 2014-07-13.

**Comment by Spinus1** [comment]
Yes, it is old: I think that the file format that srecord is managing are stable since lot of years, so srecord itself does not need frequently updates...
Some one has just opened a discussion on the forum https://sourceforge.net/p/srecord/discussion/248568/, let's see if the maintainer is still managing the project...

**Comment by Spinus1** [comment]
I've just seen that Ubuntu still provide it in universe repository: https://packages.ubuntu.com/disco/srecord

**Comment by Sherryzhang2** [comment]
I have the need to install SRecord on msys2. Any update on this request?

**Comment by Biswa96** [comment]
A srecord package has been added. Please test it. More info in `packages.msys2.org`.

**Comment by Sherryzhang2** [comment]
Could you share with me the package name?

**Comment by StarWolf3000** [comment]
It's a MINGW package: https://packages.msys2.org/search?t=binpkg&q=srecord

**Comment by Sherryzhang2** [comment]
Thanks!

**Comment by StarWolf3000** [comment]
@lazka Maybe move this over to the MINGW repo?
